napari/layers/labels/labels.py

```diff
@@ -533,6 +533,19 @@ def to_xml_list(self):
         )
         return [xml]
 
+    def _add_zarr_data(self, gp):
+        """Add layer data to zarr group."""
+        gp.array(
+            'data',
+            self.data,
+            shape=self.data.shape,
+            chunks=(128,) * len(self.data.shape),
+            dtype=self.data.dtype,
+        )
```

Comment:
Given that this is identical to image, it might be worth sharing some code between them.